mos-platform/nes-unrom/prg-rom.ld

```diff
@@ -0,0 +1,265 @@
+  FULL(prg_rom_113)
+  FULL(prg_rom_114)
+  FULL(prg_rom_115)
+  FULL(prg_rom_116)
+  FULL(prg_rom_117)
+  FULL(prg_rom_118)
+  FULL(prg_rom_119)
+  FULL(prg_rom_120)
+  FULL(prg_rom_121)
+  FULL(prg_rom_122)
+  FULL(prg_rom_123)
+  FULL(prg_rom_124)
+  FULL(prg_rom_125)
+  FULL(prg_rom_126)
+  FULL(prg_rom_127)
+  FULL(prg_rom_128)
+  FULL(prg_rom_129)
+  FULL(prg_rom_130)
+  FULL(prg_rom_131)
+  FULL(prg_rom_132)
+  FULL(prg_rom_133)
+  FULL(prg_rom_134)
+  FULL(prg_rom_135)
+  FULL(prg_rom_136)
+  FULL(prg_rom_137)
+  FULL(prg_rom_138)
+  FULL(prg_rom_139)
+  FULL(prg_rom_140)
+  FULL(prg_rom_141)
+  FULL(prg_rom_142)
+  FULL(prg_rom_143)
+  FULL(prg_rom_144)
+  FULL(prg_rom_145)
+  FULL(prg_rom_146)
+  FULL(prg_rom_147)
+  FULL(prg_rom_148)
+  FULL(prg_rom_149)
+  FULL(prg_rom_150)
+  FULL(prg_rom_151)
+  FULL(prg_rom_152)
+  FULL(prg_rom_153)
+  FULL(prg_rom_154)
+  FULL(prg_rom_155)
+  FULL(prg_rom_156)
+  FULL(prg_rom_157)
+  FULL(prg_rom_158)
+  FULL(prg_rom_159)
+  FULL(prg_rom_160)
+  FULL(prg_rom_161)
+  FULL(prg_rom_162)
+  FULL(prg_rom_163)
+  FULL(prg_rom_164)
+  FULL(prg_rom_165)
+  FULL(prg_rom_166)
+  FULL(prg_rom_167)
+  FULL(prg_rom_168)
+  FULL(prg_rom_169)
+  FULL(prg_rom_170)
+  FULL(prg_rom_171)
+  FULL(prg_rom_172)
+  FULL(prg_rom_173)
+  FULL(prg_rom_174)
+  FULL(prg_rom_175)
+  FULL(prg_rom_176)
+  FULL(prg_rom_177)
+  FULL(prg_rom_178)
+  FULL(prg_rom_179)
+  FULL(prg_rom_180)
+  FULL(prg_rom_181)
+  FULL(prg_rom_182)
+  FULL(prg_rom_183)
+  FULL(prg_rom_184)
+  FULL(prg_rom_185)
+  FULL(prg_rom_186)
+  FULL(prg_rom_187)
+  FULL(prg_rom_188)
+  FULL(prg_rom_189)
+  FULL(prg_rom_190)
+  FULL(prg_rom_191)
+  FULL(prg_rom_192)
+  FULL(prg_rom_193)
+  FULL(prg_rom_194)
+  FULL(prg_rom_195)
+  FULL(prg_rom_196)
+  FULL(prg_rom_197)
+  FULL(prg_rom_198)
+  FULL(prg_rom_199)
+  FULL(prg_rom_200)
+  FULL(prg_rom_201)
+  FULL(prg_rom_202)
+  FULL(prg_rom_203)
+  FULL(prg_rom_204)
+  FULL(prg_rom_205)
+  FULL(prg_rom_206)
+  FULL(prg_rom_207)
+  FULL(prg_rom_208)
+  FULL(prg_rom_209)
+  FULL(prg_rom_210)
+  FULL(prg_rom_211)
+  FULL(prg_rom_212)
+  FULL(prg_rom_213)
+  FULL(prg_rom_214)
+  FULL(prg_rom_215)
+  FULL(prg_rom_216)
+  FULL(prg_rom_217)
+  FULL(prg_rom_218)
+  FULL(prg_rom_219)
+  FULL(prg_rom_220)
+  FULL(prg_rom_221)
+  FULL(prg_rom_222)
+  FULL(prg_rom_223)
+  FULL(prg_rom_224)
+  FULL(prg_rom_225)
+  FULL(prg_rom_226)
+  FULL(prg_rom_227)
+  FULL(prg_rom_228)
+  FULL(prg_rom_229)
+  FULL(prg_rom_230)
+  FULL(prg_rom_231)
+  FULL(prg_rom_232)
+  FULL(prg_rom_233)
+  FULL(prg_rom_234)
+  FULL(prg_rom_235)
+  FULL(prg_rom_236)
+  FULL(prg_rom_237)
+  FULL(prg_rom_238)
+  FULL(prg_rom_239)
+  FULL(prg_rom_240)
+  FULL(prg_rom_241)
+  FULL(prg_rom_242)
+  FULL(prg_rom_243)
+  FULL(prg_rom_244)
+  FULL(prg_rom_245)
+  FULL(prg_rom_246)
+  FULL(prg_rom_247)
+  FULL(prg_rom_248)
+  FULL(prg_rom_249)
+  FULL(prg_rom_250)
+  FULL(prg_rom_251)
+  FULL(prg_rom_252)
+  FULL(prg_rom_253)
+  FULL(prg_rom_254)
```

Comment:
Yes, the mapper can (theoretically) go up to 4MB; it is officially limited to 256KB, but a 512KB variation exists in the aftermarket space, and with the remainder of the bank bits unused...